packages/editor-ui/src/plugins/i18n/locales/en.json

```diff
@@ -347,6 +347,7 @@
 	"ndv.featureRequest": "I wish this node would...",
 	"ndv.input": "Input",
 	"ndv.input.nodeDistance": "({count} node back) | ({count} nodes back)",
+	"ndv.input.nodePosition": "(Input {index})",
```

Comment:
node position is inaccurate... a more apt name might be `inputIndex`

Reply:
This locale is no longer used